Firmware/MVP/battsci.cpp

```diff
@@ -127,26 +127,38 @@ void BATTSCI_sendFrames()
       { //all cells above 3.000 volts
 
         // Battery is full. Disable Regen.   
-        if        (vCellWithESR_counts >= 39500) { //39500 = 3.9500 volts                                                    
+        if        (vCellWithESR_counts >= 40000) { //40000 = 4.0000 volts                                                    
           frameSum_87 += BATTSCI_writeByte( 0x16 );                                         //Battery SoC (upper byte)
-          frameSum_87 += BATTSCI_writeByte( 0x20 ); //80% SoC                               //Battery SoC (lower byte)
+          frameSum_87 += BATTSCI_writeByte( 0x2A ); //81% SoC                               //Battery SoC (lower byte)
           debugUSB_sendChar('8');
           //JTS2doNow: Change SoC to 81%
 
+        // Regen & Assist, no background charge   
```

Comment:
This block from lines 136 to 141 won't work properly in a 2000-2004 Insight.

Having this here will allow it to regen when the cell voltage is above 3.95V, which we don't want to happen.

75.1% is not a magic number we want to be using because a 2005-2006 Insight treats it differently from a 2000-2004.

Reply:
Thank you for pointing this out. This is correct. Do you know if 76% SoC on early Insights will cause "soft" regen lockout?